### ISSUE TITLE
Unexpected show all labels button

### ISSUE THREAD

**Issue by maciaszczykm** [kubernetes/dashboard#793]
If there are no labels `show all labels` button is displayed, but it shouldn't.

![image](https://cloud.githubusercontent.com/assets/2823399/15711341/dbeddb4c-280d-11e6-9ef1-7a1c35c90018.png)

![image](https://cloud.githubusercontent.com/assets/2823399/15711390/1734ad84-280e-11e6-839c-ab3bb88d5555.png)

**Comment by maciaszczykm** [comment]
Tested on service view, with service:

```
apiVersion: v1
kind: Service
metadata:
  name: nginx-service
spec:
  ports:
  - port: 8000 # the port that this service should serve on
    # the container on each pod to connect to, can be a name
    # (e.g. 'www') or a number (e.g. 80)
    targetPort: 80
    protocol: TCP
  # just like the selector in the replication controller,
  # but this time it identifies the set of pods to load balance
  # traffic to.
  selector:
    app: nginx
```

**Comment by bryk** [comment]
This is shown for resources that do not have labels. Can you fix this application-wide?

**Comment by maciaszczykm** [comment]
Yes, I'll fix it later.

**Comment by bryk** [comment]
@digitalfishpond Is this fixed?

**Comment by digitalfishpond** [comment]
Yes, this is fixed with #808
